File: application/models/abstract/AbstractUser.php
<?php
/**
 * Абстрактный класс пользователя
 * @author ismd
 */

abstract class AbstractUser extends PsAbstractModel {

    protected $_id;
    protected $_login;
    protected $_password;
    protected $_password1;
    protected $_email;
    protected $_info;
    protected $_site;
    protected $_registered;

    public function toArray() {
        return array(
            'id'         => $this->id,
            'login'      => $this->login,
            'email'      => $this->email,
            'info'       => $this->info,
            'site'       => $this->site,
            'registered' => $this->registered,
        );
    }

    public function setId($value) {
        $this->_id = (int)$value;
        return $this;
    }

    public function getId() {
        return $this->_id;
    }

    public function setLogin($value) {
        $this->_login = (string)$value;
        return $this;
    }

    public function getLogin() {
        return $this->_login;
    }

    public function setPassword($value) {
        $this->_password = (string)$value;
        return $this;
    }

    public function getPassword() {
        return $this->_password;
    }

    public function setEmail($value) {
        $this->_email = (string)$value;
        return $this;
    }

    public function getEmail() {
        return $this->_email;
    }

    public function setInfo($value) {
        $this->_info = (string)$value;
        return $this;
    }

    public function getInfo() {
        return $this->_info;
    }

    public function setSite($value) {
        $this->_site = (string)$value;
        return $this;
    }

    public function getSite() {
        return $this->_site;
    }

    public function setRegistered($value) {
        $this->_registered = $value;
        return $this;
    }

    public function getRegistered() {
        return $this->_registered;
    }
}

File: bin/server.common/src/game/models/common/AbstractUser.java
package game.models.common;

/**
 * @author ismd
 */
public class AbstractUser {

    private int _id;
    private String _login;
    private String _password;
    private String _password1;
    private String _email;
    private String _info;
    private String _site;
    private String _registered;

    public int getId() {
        return _id;
    }

    public AbstractUser setId(int id) {
        this._id = id;
        return this;
    }

    public String getLogin() {
        return _login;
    }

    public AbstractUser setLogin(String login) {
        this._login = login;
        return this;
    }

    public String getPassword() {
        return _password;
    }

    public AbstractUser setPassword(String password) {
        this._password = password;
        return this;
    }

    public String getPassword1() {
        return _password1;
    }

    public AbstractUser setPassword1(String password1) {
        this._password1 = password1;
        return this;
    }

    public String getEmail() {
        return _email;
    }

    public AbstractUser setEmail(String email) {
        this._email = email;
        return this;
    }

    public String getInfo() {
        return _info;
    }

    public AbstractUser setInfo(String info) {
        this._info = info;
        return this;
    }

    public String getSite() {
        return _site;
    }

    public AbstractUser setSite(String site) {
        this._site = site;
        return this;
    }

    public String getRegistered() {
        return _registered;
    }

    public AbstractUser setRegistered(String registered) {
        this._registered = registered;
        return this;
    }
}
